runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/process.json

```diff
@@ -308,18 +308,20 @@
     "Process.Details.Modeler.Actions.SharePointExcel.UpdateRange.Cell": "Specific cell",
     "Process.Details.Modeler.Actions.SharePointExcel.UpdateRange.Values": "Values [variable]",
     "Process.Details.Modeler.Actions.SharepointExcel.OpenFile.System": "System",
-    "Process.Details.Modeler.Actions.SharepointExcel.OpenFile.Label": "Open file",
+    "Process.Details.Modeler.Actions.SharepointExcel.OpenFile.Label": "Open workbook",
     "Process.Details.Modeler.Actions.SharepointExcel.OpenFileFromSite.Label": "Open file from site and create session",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.SiteRelativePath": "Site name",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.SiteRelativePath.Tooltip": "It follows the hostname in the URL and starts with 'sites/.../'",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.ListName": "List name",
-    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.FilePath": "Full path to file",
+    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.FilePath": "Path to file",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.WorksheetName": "Worksheet name",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromSite.PersistChanges": "Persist changes",
     "Process.Details.Modeler.Actions.SharepointExcel.OpenFileFromRoot.Label": "Open file from root and create session",
-    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath": "Full path to file",
+    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath": "Path to file",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.WorksheetName": "Worksheet name",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.PersistChanges": "Persist changes",
+    "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath.Info": "Refers to the location path, where the starting point is the folder, where the bot lands after login to the account, e.g. folderName/fileName.ext",
```

Comment:
Much more clearer now :). Suggestion: since it is a SP excel action I think we can leave .xlsx as an extension. Same for Polish version.